packages/bits/src/lib/wizard-v2/wizard-icon.directive.ts

```diff
@@ -0,0 +1,26 @@
+import {Directive, Input, TemplateRef} from "@angular/core";
+import {StepState} from "@angular/cdk/stepper";
+
+/** Template context available to an attached `NuiWizardIcon`. */
+export interface NuiWizardIconContext {
+    index: number;
+    active: boolean;
+    optional: boolean;
+}
+
+export interface MatStepperIconContext {
+    index: number;
+    active: boolean;
+    optional: boolean;
+}
```

Comment:
Do we need both of these since they're identical?

Reply:
No, we don't need both. I forgot to remove